.github/workflows/golangci-lint.yml

```
@@ -24,8 +24,11 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-
+      - name: install golang
```

nit: capitalize step to align with others

```suggestion
      - name: Install Go
```

---

done.